src/test/java/test/github2440/TimeoutTest.java

```diff
@@ -0,0 +1,21 @@
+package test.github2440;
```

Comment:
Also please move the tests to the package `test.timeout` since this package already houses timeout related tests.

Reply:
done.